main/res/values/strings.xml

```diff
@@ -1651,13 +1614,6 @@
         <item quantity="other">Add this cache to favorites (%d remaining)</item>
     </plurals>
 
-    <!-- distance units -->
-    <string name="unit_m">m</string>
-    <string name="unit_km">km</string>
-    <string name="unit_ft">ft</string>
-    <string name="unit_yd">yd</string>
-    <string name="unit_mi">mi</string>
-
```

Comment:
For these units I am unsure, whether they could be different in a different alphabet, like e.g. greek, cyrillic, etc?

Reply:
Ok, will revert this part - most languages use the original abbreviations, but some languages have translated the full names.